hail/www/navbar.html

```diff
@@ -51,6 +51,10 @@
       <li id="code" class="nav-item">
         <a href="https://github.com/hail-is/hail">CODE</a>
       </li>
+      <li id='workshops' class="nav-item">
+        <a href="https://workshop.hail.is" target="_blank">WORKSHOPS</a>
```

Comment:
Ah, nice.  I feel like this shouldn't open in new.

Reply:
yeah, agree

Reply:
Done!